frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineSuccessRatioDonut.tsx

```diff
@@ -68,12 +68,13 @@ const PipelineSuccessRatioDonut: React.FC<PipelineMetricsGraphProps> = ({
       ...obj,
       count: obj.y,
       sortOrder,
+      x: _.capitalize(String(obj.x)),
       y: parseFloat(percentage),
       name: `${obj.x}: ${percentage}%`,
     };
   });
   let successTimeSeriesObj = finalArray.reduce((acc, obj) => {
-    if (obj.x !== 'success') return acc;
+    if (obj.x !== 'Success') return acc;
```

Comment:
Shouldn't we use an enum for these checks?? something like `Status.Success`

Reply:
I was not able to find an existing enum for pipeline graph axes. Should I create one?
Not sure if it's required since we only check for the `Success` axis here.

Reply:
not very necessary, but it would be really good, if you can store this string `Success` in a variable as it is being used in multiple places, IMO

Reply:
Updated.